docs/source/tips-and-tricks.rst

```diff
@@ -25,50 +25,71 @@ Rxvt-unicode
 Terminus font and urxvt
 -----------------------
 
-The Terminus fonts does not have the powerline glyphs and unless someone submits 
-a patch to the font author, it is unlikely to happen.  However, Andre Klärner 
+The Terminus fonts does not have the powerline glyphs and unless someone submits
+a patch to the font author, it is unlikely to happen.  However, Andre Klärner
 came up with this work around: In your ``~/.Xdefault`` file add the following::
 
   urxvt*font: xft:Terminus:pixelsize=12,xft:Inconsolata\ for\ Powerline:pixelsize=12
 
-This will allow urxvt to fallback onto the Inconsolata fonts in case it does not 
+This will allow urxvt to fallback onto the Inconsolata fonts in case it does not
 find the right glyphs within the terminus font.
 
 Source Code Pro font and urxvt
 ------------------------------
 
-Much like the terminus font that was mentioned above, a similar fix can be 
+Much like the terminus font that was mentioned above, a similar fix can be
 applied to the Source Code Pro fonts.
 
 In the ``~/.Xdefaults`` add the following::
 
     URxvt*font: xft:Source\ Code\ Pro\ Medium:pixelsize=13:antialias=true:hinting=true,xft:Source\ Code\ Pro\ Medium:pixelsize=13:antialias=true:hinting=true
```

Comment:
Seeing this i'm asking myself if it is intended, that the same font is used twice? That is not really a fallback option, is it?